src/controllers/v2/auth.controller.js

```
@@ -0,0 +1,167 @@
+        .object({
+          phone: z
+            .string({ required_error: "Phone number is required." })
+            .length(10, "Phone number must be exactly 10 digits long."),
+          dateOfBirth: z
+            .string({ required_error: "Date of birth is required." })
+            .regex(
+              /^\d{2}-\d{2}-\d{4}$/,
+              "Invalid date format. Use DD-MM-YYYY."
+            ),
+        })
+        .parse(req.body);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)